performance_test/thread.py

```diff
@@ -0,0 +1,92 @@
+import time
+import os
+
+from google.cloud import storage
+
+RAW_BUCKET = "sie-raw-images"
+STIMULI_BUCKET = "sie-stimuli-images"
+
+
+class thread:
```

Comment:
class `Thread` and please rename this and/or the file to something more descriptive

Reply:
also this is not really a thread right? This is just defining a class and its methods, then passing the callable to `ThreadPoolExecutor` where it will be run in threads

Reply:
Is Pipeline a good name? I am bad in coming up with names. Any suggestion?

Reply:
I am gonna name it PipelineRunnable

Reply:
haha im bad with names too. Sure, let's use this for now 🥇